flow/layers/physical_model_layer.cc

```diff
@@ -9,6 +9,11 @@
 
 namespace flow {
 
+PhysicalModelLayer::PhysicalModelLayer() {
+  isRect_ = false;
+  frameRRect_ = SkRRect::MakeEmpty();
```

Comment:
No need to do this explicitly - SkRRect's default constructor will make it empty